draft-ietf-quic-qpack.md

```diff
@@ -742,19 +747,15 @@ the existing entry is represented as an integer with a 5-bit prefix.
 {:#fig-index-with-duplication title="Duplicate"}
 
 The existing entry is re-inserted into the dynamic table without resending
-either the name or the value. This is useful to mitigate the eviction of older
-entries which are frequently referenced, both to avoid the need to resend the
-header and to avoid the entry in the table blocking the ability to insert new
-headers.
+either the name or the value. This is useful to avoid adding a reference to an
+older entry, which might block inserting new entries.
 
 
 ## Decoder Instructions {#decoder-instructions}
 
-Decoder instructions provide information used to ensure consistency of the
-dynamic table. They are sent from the decoder to the encoder on a decoder
-stream; that is, the server informs the client about the processing of the
```

Comment:
This discussion of client and server seems out of place.  If it is really necessary, Section 2 might be a better place to talk about the two contexts and how both endpoint needs to have an encoder and a decoder.